src/h2olog/h2olog.cc

```diff
@@ -190,21 +217,26 @@ int main(int argc, char **argv)
     }
 
     int debug = 0;
+    int list_and_exit = 0;
     FILE *outfp = stdout;
     std::vector<std::string> event_type_filters;
     std::vector<std::string> response_header_filters;
     int c;
     pid_t h2o_pid = -1;
-    while ((c = getopt(argc, argv, "hdp:t:s:w:")) != -1) {
+    while ((c = getopt(argc, argv, "hdp:t:s:w:l")) != -1) {
         switch (c) {
         case 'p':
             h2o_pid = atoi(optarg);
             break;
         case 't':
-            event_type_filters.push_back(optarg);
+            for (const auto &item : split_str(',', optarg)) {
+                event_type_filters.push_back(item);
+            }
```

Comment:
Would you mind elaborating why we need to start using `,` as a separator? Users can specify multiple names already by repeating the option.

Reply:
Just because I prefer `-t h2o:foo,quicly:bar` because it is shorter than `-t h2o:foo -t quicly:bar`.

Reply:
Thank you for elaborating. If that's the only reason, I'd prefer not making this change. It's going to be a slippery slope of people wanting more "nifty" features as they think. As an example, why don't we remove white spaces after `,`? People tend to prefer having spaces after `,`, don't they?

Seriously, the operating system does provide the interface for providing a list of arguments (by repeating the option name). Let's rely on that.

Reply:
It's the only reason! And as you said, I was not sure if I should trim whitespaces or not, meaning it was likely to confuse users.

Removed at c40034bcf6e27a1e554de5044e88394cc9dc6ee7